data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
 			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
 			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
 				decline
+
+
+
+mission "Lost Dog"
+	name "Bring a dog to <planet>"
+	description "Bring Caesar, a dog you found lost on <source>, to their owner on <destination>."
+	to offer
+		random < 15
+	source "Calda"
+	destination "Hestia"
+	on offer
+		conversation
+			`Your walk from the <ship> takes you outside <planet>'s bustling spaceport and into the overtly welcoming and carefully inoffensive tourist-saturated streets of the spaceport town. Every path seems purpose-built to whisk you further away from your ship and deeper into the unnatural radiance of this Paradise World. The hand-crafted beauty is inescapable, a never-ending sequence of breathtaking views.`
+			`	As you walk you eventually come across an immaculate and perfectly manicured garden park. Normally there is an exorbitant fee to enter, but due to some promotion today you can enter freely.`
+			choice
+				`	(Go to the park.)`
+					goto park
+				`	(Head back to your ship.)`
+
+			`	It was a splendid walk, but you decide it's time to head back. You have more important things to do than wander around an expensive terraformed garden.`
```

Comment:
Again, "it was a splendid walk" and "you have more important things to do" imply that I'm (a) not just a tourist exploring this planet while also getting close to telling me what I think. This sounds like a sentence out of a game of interactive fiction that is trying to hint towards a specific course of action.